src/RefractAPI.cc

```diff
@@ -187,26 +187,18 @@ namespace drafter {
     refract::IElement* ParameterToRefract(const NodeInfo<snowcrash::Parameter>& parameter, ConversionContext& context)
     {
         refract::MemberElement* element = new refract::MemberElement;
-        refract::IElement *value = NULL;
-
-        // Parameter type, exampleValue, defaultValue, values
-        if (parameter.node->type == "boolean") {
-            value = ExtractParameter<bool>(parameter, context);
-        }
-        else if (parameter.node->type == "number") {
-            value = ExtractParameter<double>(parameter, context);
-        }
-        else {
-            value = ExtractParameter<std::string>(parameter, context);
-        }
-
+        refract::IElement *value = ExtractParameter<std::string>(parameter, context);
```

Comment:
I've addressed this in https://github.com/apiaryio/drafter/pull/447/commits/c65755cd0a3bfa846cf47fff0b8413c949b4e353.